### DISCUSSION TITLE
Collection panel upgrades (search, quick select, sort, group)

### DISCUSSION THREAD

**Discussion by sam-or** [darrenburns/posting#74] [Ideas]
Just discovered this project and I'm already a big fan. I have a few feature ideas that I'd love to get your thoughts on @darrenburns. I'd also like to contribute to the best of my ability if that's something you are open to?

1. I would love to be able to search using '/' to filter the requests within the Collections panel
2. If there was a way to quickly jump to a request using fuzzy selection (like [fzf](https://github.com/junegunn/fzf)) that would be awesome
3. Groping/organising requests into folders within collapsible sections
    - Auto-grouping by tag from openapi.json import

**Comment by zhangddjs** [comment]
Agree, fzf like quick jump mechanizm is quite important, without this feature, the usage is very inefficient. hope this feature can coming ASAP

**Reply by oxodao** [reply]
Especially since there is already a Ctrl+P command box, it could be just like VSCode where Ctrl+P does search and Ctrl+Shift+P opens the commands

**Reply by darrenburns** [reply]
Posting 2.4 adds `ctrl+shift+p` which opens a fuzzy search popup similar to VSCode :)

**Comment by darrenburns** [comment]
Posting 2.4 adds the first suggestion. You can press `ctrl+shift+p`, which opens a fuzzy search popup similar to VSCode :)

The 3rd suggestion is already implemented - you can freely add folders. If you change the `path` field in the "New Request" modal, Posting will create the required directory structure.
